### PR TITLE
Update docs/color page content

### PR DESCRIPTION
**Pull request by kwm14**
## Done

- Added accessibility section
- Added imagery
- Added do's and don't examples
- Check content matches copy doc [here](https://docs.google.com/document/d/1Ly43Jb5Cav3Eee0FgNUVTXSJAjB71KNMqLWHf9c5scw/edit#)

## QA

- Pull code
- Run `./run serve --watch`
- Check https://docs.vanillaframework.io/en/settings/color-settings

## Details
- Fixes https://github.com/ubuntudesign/vanilla-design/issues/339

### CODE REVIEW

File: docs/en/settings/color-settings.md [hunk 1]
```diff
@@ -116,8 +112,30 @@ When planning your pages, make sure there is an even distribution and well-balan
...
+   <div class="col-6">
+   <div class="p-notification--positive">
+   <p class="p-notification__response"><span class="p-notification__status">Do:</span>Minimal contrast ratio: 4.5</p>
+   </div>
+   <img class="p-image--bordered" src="https://assets.ubuntu.com/v1/9e60d838-basics-text-color-do.png" alt="text-color-do">
+   <img class="p-image--bordered" src="https://assets.ubuntu.com/v1/a55aa80e-basics-button-color-do.png" alt="button-color-do">
+   </div>
+  <div class="col-6">
+  <div class="p-notification--negative">
+  <p class="p-notification__response"><span class="p-notification__status">Don't:</span>Avoid using dark text that isn’t legible on dark backgrounds</p>
```
Comment (deadlight) [inline docs/en/settings/color-settings.md]: I think we should reword this is sounds like "don't avoid using..."
```suggestion
  <p class="p-notification__response"><span class="p-notification__status">Don't:</span>Use dark text that isn’t legible on dark backgrounds</p>
```

File: docs/en/settings/color-settings.md [hunk 2]
```diff
@@ -116,8 +112,30 @@ When planning your pages, make sure there is an even distribution and well-balan
...
+
+### Accessibility
 
-* [Color design](https://github.com/ubuntudesign/vanilla-design/tree/master/Color)
+It’s important for us to meet all web accessibility standards. Vanilla encourages meeting the minimum contrast ratios specified by WCAG 2.1 Level AA for text, icons and background colors.
+
+<div class="row">
+   <div class="col-6">
+   <div class="p-notification--positive">
+   <p class="p-notification__response"><span class="p-notification__status">Do:</span>Minimal contrast ratio: 4.5</p>
```
Comment (deadlight) [inline docs/en/settings/color-settings.md]: To match tone with my suggestion on the "don't" side:
```suggestion
   <p class="p-notification__response"><span class="p-notification__status">Do:</span>Use a minimum contrast ratio of 4.5</p>
```

File: docs/en/settings/color-settings.md [hunk 3]
```diff
@@ -116,8 +112,30 @@ When planning your pages, make sure there is an even distribution and well-balan
...
+  <div class="p-notification--negative">
+  <p class="p-notification__response"><span class="p-notification__status">Don't:</span>Avoid using dark text that isn’t legible on dark backgrounds</p>
+  </div>
+  <img class="p-image--bordered" src="https://assets.ubuntu.com/v1/9f98d99d-basics-text-color-don%27t.png" alt="text-color-don't">
+  <img class="p-image--bordered" src="https://assets.ubuntu.com/v1/4b8dc06e-basics-button-color-don%27t.png" alt="button-color-don't">
+  </div>
+</div>
+
+### Design
+Click [here](https://github.com/ubuntudesign/vanilla-design/tree/master/Color) to view Color design spec, which includes build MD file and PNG image.
```
Comment (deadlight) [inline docs/en/settings/color-settings.md]: We shouldn't use "Click here", links should always be semantic descriptions of where they go:
```suggestion
For more information [view the color design spec](https://github.com/ubuntudesign/vanilla-design/tree/master/Color), which includes the specification in markdown format and a PNG image.
```

**Comment by deadlight** [comment]
Here's a screenshot of the rendered page. I think we need to tweak the spacing.

image

**Comment by kwm14** [comment]
@deadlight fixed images to resolve spacing issue. Ready for re-review 👍

**Pull request merged**